polynote-frontend/polynote/ui/component/notebooklist.ts

```diff
@@ -5,27 +5,27 @@ import {
     dropdown,
     DropdownElement,
     h2,
-    helpIconButton,
+    helpIconButton, icon,
     iconButton,
     para,
     span,
     tag,
     TagElement
 } from "../tags";
 import {ServerMessageDispatcher} from "../../messaging/dispatcher";
-import {deepCopy, diffArray, getShortDate} from "../../util/helpers";
+import {deepCopy, diffArray, getHumanishDate, getShortDate} from "../../util/helpers";
```

Comment:
[nit] I think we can remove the `getShortDate` function entirely with this new formatting helper